__std__/nix/toolchain/packages/repo-root.nix

```diff
@@ -0,0 +1,7 @@
+{ inputs, cell }:
+
+inputs.nixpkgs.writeShellApplication {
+  name = "repo-root";
+  text = "git rev-parse --show-toplevel";
```

Comment:
This is so common, that within the devshell, repo root is also available as `PRJ_ROOT`.

Reply:
@blaggacao This little comment took me down a rabbit hole. 

`PRJ_ROOT` is available inside the devshell, but not outside, right?
So the question is, why would we need an equivalent to `PRJ_ROOT`(like `repo-root`) for use outside a devshell?
So that we can have "GitOps" scripts that can be run outside a devshell.

Consider this derivation: `__std__/nix/toolchain/packages/repo-root.nix`
```
{ inputs, cell }:

inputs.nixpkgs.writeShellApplication {
  name = "autobuild";

  runtimeInputs = [
    inputs.cells.toolchain.packages.repo-root
    inputs.cells.toolchain.packages.sphinx-toolchain
  ];

  text = ''
    root="$(repo-root)"
    sphinx-autobuild -j 4 -n "$root/doc" "$root/doc/_build"
  '';
}
```
This is completely standalone. 
It can be run inside and outside a devshell.
The price to pay for it is what we have to write the `runtimeInputs`, 
which are duplicated in the `__std__/nix/doc/devshells/doc-shell.nix:packages` field.


Now consider this:
```
{ inputs, cell }:

inputs.nixpkgs.writeScript "autobuild" ''
  sphinx-autobuild -j 4 -n "$PRJ_ROOT/doc" "$PRJ_ROOT/doc/_build"
''
```
Much nicer! 
But it assumes that you are inside a devshell with `sphinx-autobuild`
(Indeed in the previous example, `sphinx-autobuild` was provided by `inputs.cells.toolchain.packages.sphinx-toolchain` in the `runtimeInputs`)

Once again there is a tradeoff to be made.
Perhaps this calls for a `devshellScripts` clade(?), which everyone agrees is for scripts meant to be run inside a 
devshell.
Whereas a more general `script` organelles would be for non-devshell scripts.


Maybe the best solution is to use both "kinds" of scripts depending on what they do.
If I am certain that a script is to be run inside a devshell, i can use `writeScript` and rely on the `EVN` inside the devshell.
I might wanna put these scripts in a `devshellScripts` organelle (actually, `devshellCommands` might be better).

If instead I'm just writing a devshell-agnostic script, I will use `writeShellApplication` with an explicit `runtimeInputs`.
I might wanna put these scripts in a more general `scripts` or `jobs` organelle instead.

What are your thoughts?